pandas/tests/plotting/test_series.py

```diff
@@ -729,6 +729,26 @@ def test_dup_datetime_index_plot(self):
         s = Series(values, index=index)
         _check_plot_works(s.plot)
 
+    def test_errorbar_asymmetrical(self):
+        # GH9536
+        s = Series(np.arange(10), name="x")
+        err = np.random.rand(2, 10)
+
+        ax = s.plot(yerr=err, xerr=err)
+
+        yerr_0_0 = ax.collections[1].get_paths()[0].vertices[:, 1]
+        expected_0_0 = err[:, 0] * np.array([-1, 1])
```

Comment:
Would it be possible to check all the points here? Something like
```python
result = np.vstack([i.vertices[:, 1] for i in ax.collections[1].get_paths()])
expected = err.T * np.array([-1, 1])
```
?

Reply:
Addressed in https://github.com/pandas-dev/pandas/pull/34514/commits/2975406501f8942ded75091594f094e8d3095a2a